18-hlt-intro.md

```diff
@@ -32,15 +32,24 @@ lines are saved to the raw event in a stripped-down form. One of the things that
 is saved are all the LHCbIDs of the final state particles of a decay tree.
 LHCbIDs are unique identifiers for detector hits.
 
-We will now have a look at the LHCbIDs of some of the stored candidates. We will
-use the script we [created earlier](http://lhcb.github.io/first-analysis-steps/05-interactive-dst.html) as a starting
-point, and the file you [downloaded before](http://lhcb.github.io/first-analysis-steps/05-files-from-grid.html). Fire up
-your favourite editor, open the script and save a copy to work on as
-`tistos.py`. There are a few things in the script that we don't need and can be
-removed, such as the print_decay and decay finder tools.
+A new feature in Run II is the so-called Turbo stream. Since the reconstruction
+available in HLT2 is the same as the offline reconstruction, physics analysis
+can be done with the candidates created in HLT2. If a line is configured to be a
+Turbo line, all information on the candidates that it selects is stored in the
+raw event. These candidates can be resurrected later by the Tesla application and
+written to a microDST.
```

Comment:
Could add

> This is similar to stripping streams that go to microDST, where only candidates that are used in passing selections are available to analysts. The Turbo stream is different because information that is not saved is lost forever.

(Maybe slightly less dramatically.)